Automattic/themes
# Color Annotations: Check for the existence of color annotations in a more robust way

#### Changes proposed in this Pull Request:
On .org sites get_theme_mod returns `false` when there is nothing set, but on Simple and Atomic sites it returns an array. This makes the check for the existence of color annotations more robust, so that it works if no color annotations are set.

#### Testing instructions
- Create a simple site with no color annotations and visit the editor - you won't see any editor palettes.
- Or run wpcomsh locally and see the same thing.

#### Related issue(s):
Fixes #3231
